--- a/packages/libbz2/build.sh
+++ b/packages/libbz2/build.sh
@@ -4,8 +4,8 @@ TERMUX_PKG_LICENSE="BSD"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=1.0.8
 TERMUX_PKG_REVISION=8
-TERMUX_PKG_SRCURL=https://fossies.org/linux/misc/bzip2-${TERMUX_PKG_VERSION}.tar.xz
-TERMUX_PKG_SHA256=47fd74b2ff83effad0ddf62074e6fad1f6b4a77a96e121ab421c20a216371a1f
+TERMUX_PKG_SRCURL=https://sourceware.org/pub/bzip2/bzip2-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=ab5a03176ee106d3f0fa90e381da478ddae405918153cca248e682cd0c4a2269
 TERMUX_PKG_BREAKS="libbz2-dev"
 TERMUX_PKG_REPLACES="libbz2-dev"
 TERMUX_PKG_ESSENTIAL=true
